README.md

```diff
@@ -12,3 +12,59 @@ Almond is part of Open Thing Platform, a research project led by
 prof. Monica Lam, from Stanford University.  You can find more
 information at <https://thingpedia.stanford.edu/about>.
 
+## Installation
+
+This assumes that you have [node](https://github.com/nodejs/node) and [yarn](https://github.com/yarnpkg/yarn) installed, with at least **2 GB** of available diskspace. Please be sure to be on a **fast internet connection** before going through the installation.
```

Comment:
Do we really need 2GB for Almond Server? What's so big about it?

Reply:
Oh I think this assumed the need for Mimic. Can remove when the chat beta replaces the old chat entirely.

Reply:
It was mimic that took a lot of space. Can remove